### PR TITLE
[tests-only] Extend API test coverage for MOVE between spaces using file-id

### PR DESCRIPTION
**Pull request by saw-jan**
## Description
Extends MOVE API test coverage:

**Using file-id**
| from       | from (role)           | to                  | to (role)             | result         | API coverage       |
| ---------- | --------------------- | ------------------- | --------------------- | -------------- | ------------------ |
| project    | manager/editor        | project(same)       |                       | 201 :+1:       | :heavy_check_mark: |
| project    | viewer                | project(same)       |                       | 403 :+1:       | :heavy_check_mark: |
| project    | manager/editor        | project             | manager/editor        | 502 :question: | :heavy_check_mark: |
| project    | manager/editor        | project             | viewer                | 403 :+1:       | :heavy_check_mark: |
| project    | viewer                | project             | manager/editor/viewer | 403 :+1:       | :heavy_check_mark: |
| project    | manager/editor        | personal            |                       | 502 :question: | :heavy_check_mark: |
| project    | viewer                | personal            |                       | 403 :+1:       | :heavy_check_mark: |
| project    | manager/editor/viewer | share jail          | all/editor/viewer     | 403 :+1:       | :heavy_check_mark: |
| personal   |                       | personal(same)      |                       | 201 :+1:       | :heavy_check_mark: |
| personal   |                       | personal(subfolder) |                       | 201 :+1:       | :heavy_check_mark: |
| personal   |                       | project             | manager/editor        | 502 :question: | :heavy_check_mark: |
| personal   |                       | project             | viewer                | 403 :+1:       | :heavy_check_mark: |
| personal   |                       | share jail          | all/editor/viewer     | 403 :+1:       | :heavy_check_mark: |
| share jail | all/editor            | share jail(same)    |                       | 403 :x:        | :heavy_check_mark: |
| share jail | viewer                | share jail(same)    |                       | 403 :+1:       | :heavy_check_mark: |
| share jail | all/editor            | personal            |                       | 502 :question: | :heavy_check_mark: |
| share jail | viewer                | personal            |                       | 403 :+1:       | :heavy_check_mark: |
| share jail | all/editor/viewer     | share jail          | all/editor/viewer     | 403 :+1:       | :heavy_check_mark: |
| share jail | all/editor            | project             | manager/editor        | 502 :question: | :heavy_check_mark: |
| share jail | all/editor            | project             | viewer                | 403 :+1:       | :heavy_check_mark: |
| share jail | viewer                | project             | manager/editor/viewer | 403 :+1:       | :heavy_check_mark: |

## Related Issue
- https://github.com/owncloud/ocis/issues/8121
- https://github.com/owncloud/ocis/issues/6737

Coverage for:
- https://github.com/owncloud/ocis/issues/7617
- https://github.com/owncloud/ocis/issues/8124
- https://github.com/owncloud/ocis/issues/8125
- https://github.com/owncloud/ocis/issues/8116
- https://github.com/owncloud/ocis/issues/7618

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:
- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: tests/acceptance/features/apiSpacesShares/moveSpaces.feature [hunk 1]
```diff
@@ -59,13 +59,13 @@ Feature: move (rename) file
...
     And for user "Alice" the space "Project1" should contain these entries:
       | project1.txt |
     And for user "Alice" the space "Project2" should not contain these entries:
       | project1.txt |
     Examples:
-      | from_role | to_role | https_status_code |
+      | from_role | to_role | http-status-code  |
       | manager   | manager | 502               |
       | editor    | manager | 502               |
       | manager   | editor  | 502               |
```
Comment (ScharfViktor) [inline tests/acceptance/features/apiSpacesShares/moveSpaces.feature:71]: according https://github.com/owncloud/ocis/issues/8116 issue we expect here 403?

File: tests/acceptance/features/apiSpacesShares/moveSpaces.feature [hunk 2]
```diff
@@ -133,13 +133,13 @@ Feature: move (rename) file
...
+    Then the HTTP status code should be "<http-status-code>"
     And for user "Alice" the space "Personal" should contain these entries:
       | personal.txt |
     And for user "Alice" the space "Project" should not contain these entries:
       | personal.txt |
     Examples:
-      | role    | https_status_code |
+      | role    | http-status-code  |
       | manager | 502               |
       | editor  | 502               |
```
Comment (ScharfViktor) [inline tests/acceptance/features/apiSpacesShares/moveSpaces.feature:144]: 403?

File: tests/acceptance/features/apiSpacesShares/moveSpaces.feature [hunk 3]
```diff
@@ -85,13 +85,13 @@ Feature: move (rename) file
...
+    Then the HTTP status code should be "<http-status-code>"
     And for user "Alice" the space "Project" should contain these entries:
       | project.txt |
     And for user "Alice" the space "Personal" should not contain these entries:
       | project.txt |
     Examples:
-      | role    | https_status_code |
+      | role    | http-status-code  |
       | manager | 502               |
       | editor  | 502               |
```
Comment (ScharfViktor) [inline tests/acceptance/features/apiSpacesShares/moveSpaces.feature:96]: 403?

File: tests/acceptance/features/apiSpacesShares/moveSpaces.feature (hunk 1 above)
Comment (saw-jan) [reply tests/acceptance/features/apiSpacesShares/moveSpaces.feature:71]: yeah, it should be 403? Should I expect 403 and put them in expected failure list?

Comment (ScharfViktor) [reply tests/acceptance/features/apiSpacesShares/moveSpaces.feature:71]: I think yes

Comment (saw-jan) [reply tests/acceptance/features/apiSpacesShares/moveSpaces.feature:71]: okey thanks!

**Review by ScharfViktor — APPROVED** [review]

**Pull request merged**